AppCenterPush/AppCenterPushTests/MSPushTests.m

```diff
@@ -865,4 +869,44 @@ - (void)testUserIdContextDelegateCalledOnUserIdChanged {
   [pushMock stopMocking];
 }
 
+- (void)testServiceNotificationPushAppDelegateCallback {
+
+  // If
+  id pushDelegateMock = OCMProtocolMock(@protocol(MSPushDelegate));
+
+  // Notification custom data
```

Comment:
Not blocking but just a nitpick - we usually end comments with `.` (except for `// If`, `// Then`, and `// When`